src/components/__tests__/SentimentAnalysisGraph.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import SentimentAnalysisGraph from './../SentimentAnalysisGraph';
+import {shallow} from 'enzyme';
+
+describe('<SentimentAnalysisGraph />', () => {
+	it('Sum of positive value and negative value should be 100 with provided props', () => {
+		const wrapper = shallow(<SentimentAnalysisGraph posValue={60} negValue={40}/>);
+		const posValue = wrapper.node.props.children[1].props.data[0].y;
+		const negValue = wrapper.node.props.children[1].props.data[1].y;
+		expect(posValue + negValue).toEqual(100);
+	});
+
+	it('Default positive value and negative value shoule be 90 and 10 respectively', () => {
```

Comment:
Minor thing for next time, don't worry about fixing it now:
When you're writing Jasmine-style tests, you should write the `it` part as though `it` is part of the sentence. For example:
`it('should have a positive value of 90 and a negative value of 10', () => {...})`
It makes code and test results in the console easier to read